lib/proposed/character.rb

```diff
@@ -0,0 +1,13 @@
+class Character
```

Comment:
To jest model character, tzw. model anemiczny (nie ma żadnych zachowań). Modele powinny być jak najprostsze i nie wykonywać żadnych zachowań wewnątrz. On ma tylko publiczne pola(atrybuty), które go opisują i cechę `is_human?`, która zwraca jedynie `true` i `false`. Model mówi o sobie, a nic nie wykonuje.